openlibrary/core/yearly_reading_goals.py

```diff
@@ -8,14 +8,28 @@ class YearlyReadingGoals:
 
     # Create methods:
     @classmethod
-    def create(cls, username: str, year: int, target: int):
+    def create(cls, username: str, year: int, target: int, current: int = 0):
```

Comment:
what does current mean here?

Reply:
The current amount of books that are already read.

Reply:
Also, I'm wondering if it makes sense to even have a `current` column in the table.  It will probably be difficult to keep it updated.  I added a new function to BookshelvesEvents that gets distinct copies of works read in a year, and am using that instead of the `current` count.  This seems much easier than what I planned to do:
```
- On new check-in:
    - Capture the year as `year`
    - Insert new check-in into  bookshelves_events table
    - Update yearly_reading_goals where year = `year` for user
- On check-in update or delete:
    - Capture old year as `old_year`
    - Capture year as `year`
    - if `year` and `old_year` are different, continue
    - Update bookshelves_events
    - Update yearly_reading_goals entries for `year` and `old_year`
 ```

Reply:
This seems like a reasonable path forward, though happy to proceed with what we have in this PR to unblock our end of year push